CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Braintree Android SDK Release Notes
 
+## Unreleased
+* BraintreeCore
+  * Update pinned certificates used by `BraintreeGraphQLClient`
```

Comment:
`BraintreeGraphQLClient` is internal so not sure if this is confusing for merchants. Do we want a more generic update pinned certificates statement or is this helpful for our future selves?

Reply:
I went back and forth on this and am definitely open to the best way to surface this.  

I wouldn't normally put an internal change like this in the changelog, but since it's an extremely important change (since it will eventually impact tokenization for version prior) that we might want/need to point to directly and make discoverable, I figured this was worth the specific call out.